Server/mods/deathmatch/logic/CPlayerStats.h

```diff
@@ -29,10 +28,10 @@ class CPlayerStats
 
     bool                           GetStat(unsigned short usID, float& fValue);
     void                           SetStat(unsigned short usID, float fValue);
-    vector<sStat*>::const_iterator IterBegin() { return m_List.begin(); }
-    vector<sStat*>::const_iterator IterEnd() { return m_List.end(); }
+    std::vector<sStat*>::const_iterator IterBegin() { return m_List.begin(); }
+    std::vector<sStat*>::const_iterator IterEnd() { return m_List.end(); }
     unsigned short                 GetSize() { return static_cast<unsigned short>(m_List.size()); }
```

Comment:
this is misformatted now. could you move IterBegin and IterEnd underneath GetSize (leave a blank line in between) and then reformat the struct?

Reply:
I left it like that, because I thought once you said that I should leave these things up to the formatter. But I might remember it incorrectly.

Reply:
try formatting selection, you'll see it's not in line with the formatter